### ISSUE TITLE
Error on CAD file comparison

### ISSUE THREAD

**Issue by ndem0** [mathLab/PyGeM#205]
```
======================================================================

FAIL: test_ffd_iges_pipe_mod_through_files (test_ffdcad.TestFFDCAD)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/mathLab/PyGeM/tests/test_ffdcad.py", line 24, in test_ffd_iges_pipe_mod_through_files

    self.assertMultiLineEqual(ref[i], cre[i])

AssertionError: '9.29[16 chars]810629323,1.290550492E+03,9.343553199E+03,   0000005P0000794\n' != '9.29[16 chars]810629324,1.290550492E+03,9.343553199E+03,   0000005P0000794\n'

- 9.299036204E+03,606.810629323,1.290550492E+03,9.343553199E+03,   0000005P0000794

?                             ^

+ 9.299036204E+03,606.810629324,1.290550492E+03,9.343553199E+03,   0000005P0000794

?                             ^

======================================================================

FAIL: test_ffd_iges_pipe_mod_through_topods_shape (test_ffdcad.TestFFDCAD)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/mathLab/PyGeM/tests/test_ffdcad.py", line 41, in test_ffd_iges_pipe_mod_through_topods_shape

    self.assertMultiLineEqual(ref[i], cre[i])

AssertionError: '-134.222119722,1.118290358E+03,8.227867561E+03,-247.97[24 chars]35\n' != '-134.222119721,1.118290358E+03,8.227867561E+03,-247.97[24 chars]35\n'

- -134.222119722,1.118290358E+03,8.227867561E+03,-247.970135629,   0000005P0000735

?              ^

+ -134.222119721,1.118290358E+03,8.227867561E+03,-247.970135629,   0000005P0000735

?              ^

----------------------------------------------------------------------

Ran 2 tests in 0.303s
```

**Comment by andreamola** [comment]
I checked, and I believe this has been fixed, already.
My memory is back, but the lines in which the error occurs are not there any more. Instead, there is a check that the numbers in the iges files are equal up to the 6th figure. So, I'm qite sure this is solved, but maybe double check it.

**Comment by ndem0** [comment]
Fixed in #206
